.github/actions/ospray-sb-install-dep/action.yml

```diff
@@ -48,7 +48,7 @@ runs:
         -DCMAKE_BUILD_TYPE=Release
         -DCMAKE_INSTALL_PREFIX:PATH=../ospray_install
         -DDOWNLOAD_ISPC=ON
-        -DDOWNLOAD_TBB=ON
+        -DDOWNLOAD_TBB=OFF
         -DINSTALL_DEPENDENCIES=ON
         -DINSTALL_IN_SEPARATE_DIRECTORIES=OFF
```

Comment:
-DCMAKE_PREFIX_PATH:PATH=$(pwd)/../install/

Reply:
It's not working because the variable does not propagate through the subprojects of the superbuild.  
I'll try to use the env variable TBB_ROOT sued in their FindTBB.cmake script.